app/helpers/messages_helper.rb

```diff
@@ -1,2 +1,10 @@
 module MessagesHelper
+
+  def personal_address
+    unless current_user.name.blank?
+      address = current_user.name
+    else 
+      address = current_user.email
```

Comment:
You're trying to return either the user name or email, so why do you need to assign it to a variable?